### DISCUSSION TITLE
different image in forge and swarm

### DISCUSSION THREAD

**Discussion by kalle07** [mcmonkeyprojects/SwarmUI#376] [Q&A]
again ...
Maybe you discuss the topic on Discord ?!? (i dont)
Unfortunately I can't do it in compfy with a workflow!

so first 
Forge
![image(13)](https://github.com/user-attachments/assets/d19e6767-cf01-4d8f-8fde-81bd6f02edb9)

swarmui
![1208-10-29](https://github.com/user-attachments/assets/b2477dec-ece8-4162-ae6b-50a9b0932e50)

(meta-data inside PNG)
i only can make an x/y/z plot in forge

so I changed the cfg and the cfg scale slightly
![xyz_grid-0000-2405129555](https://github.com/user-attachments/assets/6a35ad22-c412-4256-b41b-48db5cc31604)

![xyz_grid-0001-2405129555](https://github.com/user-attachments/assets/400dc99b-3b9f-4fc8-8580-63632298b522)

so for me its like you have in forge and swarm a different model ?!?

**Comment by kwal559** [comment]
not sure purpose of generating same pic twice, but, i get bad images at distilled CFG 0.1-1.5.. However 0.0 seems to generate great images.  I think this sliding scale may be implemented differently per user interface.. InvokeAI i believe shows 2.0 - 7.0 (or there about)

**Reply by mcmonkey4eva** [reply]
Distilled CFG doesn't exist, it's a mislabeling of Flux Guidance Scale, and it's a direct parameter input to the model. It's possible invokeai does something weird, but most UIs just take the number you give and feeds it into the model. They recommend ~3.5 as the default, but lower values in the range of ~2 can be great. <= 1 is chaotic and shouldn't work, that's asking the model to given invalid output.

**Comment by kalle07** [comment]
it works well for an SDXL and a Pony model ... same parameters same image

hmmm, why not FLUX ?
